### ISSUE TITLE
C2ME Fabric crashes on dedicated server because Connector tries to load some client-only JiJ mod.

### ISSUE THREAD

**Issue by heipiao233** [Sinytra/Connector#924]
### Mod Name

C2ME Fabric

### Mod Homepage

https://modrinth.com/mod/c2me-fabric

### Minecraft version

1.20.1

### Describe the bug

It crashes.

### Steps to reproduce

1. Install Minecraft Server, Connector and C2ME fabric mod.
2. Launch it.

### Logs

https://gist.github.com/heipiao233/f337fad5c024525399f335c08111a9af

### Additional context

This is different from prev issues.

**Comment by coco875** [comment]
duplicate of https://github.com/Sinytra/Connector/issues/88

**Comment by heipiao233** [comment]
@Su5eD This is different from #88.

**Comment by coco875** [comment]
Yeah but the first problem is it doesn't work without disable some config so he will fix also this problem at the same time he makes the compat for c2me
